pkg/bloombuild/builder/builder.go

```diff
@@ -110,12 +111,33 @@ func (b *Builder) stopping(_ error) error {
 }
 
 func (b *Builder) running(ctx context.Context) error {
+	// Retry if the connection to the planner is lost.
+	retries := backoff.New(context.Background(), b.cfg.BackoffConfig)
```

Comment:
What's the reason for creating a new context instead of using `ctx`?

Reply:
Changed. Thank you!